src/modules/dashboard/components/ExpenditurePage/Streaming/LockedStreaming.css

```diff
@@ -0,0 +1,64 @@
+@value paddingTop: 56px;
+@value lockedPaddingTop: 75px;
+@value paddingRightLeftBottom: 54px;
+
+.container {
+  padding: paddingTop paddingRightLeftBottom paddingRightLeftBottom;
+}
+
+.container i {
+  /* The negative margin is used here to adjust the position of the caret icon */
+  margin-bottom: -3px;
+}
```

Comment:
This doesn't seem to be needed as there are no icons in the locked state.

Reply:
it is removed